docs/source/alfacase_definitions/CaseOutputDescription.txt

```diff
@@ -5,22 +5,22 @@
     .. parsed-literal::
 
         class CaseOutputDescription
-            trends: \ :class:`List <typing.List>`\[\ :class:`TrendOutputDescription <TrendOutputDescription>`\]
-            trend_frequency: \ :class:`Scalar <barril.units.Scalar>`\ |space|
-            profiles: \ :class:`List <typing.List>`\[\ :class:`ProfileOutputDescription <ProfileOutputDescription>`\]
-            profile_frequency: \ :class:`Scalar <barril.units.Scalar>`\ |space|
+            trends: \ :class:`List <typing.List>`\[\ :class:`TrendOutputDescription <TrendOutputDescription>`\] = []
```

Comment:
This was auto-generated correct? Can you split the commit where you change the generator from the auto-generation, to make it easier for us to review it?

Reply:
Sure

This commit changes the generator:  9dd2855
This commit updates the alfacase-definitions: 3f42885